src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        Random random = new Random();
+        int nextInt = random.nextInt(100);
```

Comment:
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants)
All magic numbers in your code should be constants.

Reply:
Thanks, good advice, I will definitely use it